src/common/constants.lua

```diff
@@ -2,5 +2,11 @@ local constants = {}
 
 constants.ARWEAVE_ID_REGEXP = "^[a-zA-Z0-9-_]{43}$"
 constants.INVALID_ARWEAVE_ID_MESSAGE = "Invalid Arweave ID"
+constants.MAX_UNDERNAME_LENGTH = 61
+constants.UNDERNAME_REGEXP = "^(?:@|[a-zA-Z0-9][a-zA-Z0-9-_]{0,"
+	.. (constants.MAX_UNDERNAME_LENGTH - 2)
+	.. "}[a-zA-Z0-9])$"
+constants.MIN_TTL_SECONDS = 900
```

Comment:
Note to self to determine why we'd have TTLs at all here.

Reply:
agreed - not needed

Reply:
removed